alire.toml

```diff
@@ -19,3 +24,12 @@ prepend = "${CRATE_ROOT}/.objs/compiler"
 
 [build-switches]
 "*".ada_version = "Ada12"
+development.optimization = ["-O0"]
```

Comment:
Just out of curiosity, do you override the default used by Alire (`-Og`) for a particular reason?

Reply:
I found debugging difficult with `-Og` sometimes. The compiler eliminates some variables and gdb can't show their values.